packages/libs/package.json

```diff
@@ -63,9 +63,9 @@
     "prepack": "turbo run build"
   },
   "dependencies": {
-    "@audius/fixed-decimal": "*",
+    "@audius/fixed-decimal": "^0.0.20",
     "@audius/hedgehog": "3.0.0-alpha.0",
-    "@audius/spl": "*",
+    "@audius/spl": "^0.0.27",
```

Comment:
does this mean we need to manually update these versions when major/minor versions change?

Reply:
Yes! Although I think `changesets` will take care of that once/if we add it